src/nvim/drawline.c

```diff
@@ -368,7 +368,7 @@ static int draw_virt_text_item(buf_T *buf, int col, VirtText vt, HlMode hl_mode,
 
 // TODO(bfredl): integrate with grid.c linebuf code? madness?
 static void draw_col_buf(win_T *wp, winlinevars_T *wlv, const char *text, size_t len, int attr,
-                         bool vcol)
+                         colnr_T *fold_vcol, bool vcol)
```

Comment:
```suggestion
                         const colnr_T *fold_vcol, bool vcol)
```

Reply:
Also I wonder if the `vcol` parameter name should now be renamed to `inc_vcol` to avoid confusion.